docs/src/rsh/errors/index.md

```diff
@@ -988,6 +988,15 @@ const o = {
 
 You can fix this by using a static string as the key.
 
+```reach
+load: ./hs/t/y/dynamic_pay.rsh
```

Comment:
```suggestion
load: /hs/t/y/dynamic_pay.rsh
```

Inconsistent paths like this mess up programs that try to programmatically add things like `md5`s, [for example](https://github.com/reach-sh/reach-lang/pull/1028/commits/e43c713be0108e277b69b3d3107436b956356eb8#diff-8ea55241077b0f25bb9381a8c20436c61c524ef0e5e6eda3717991587cc01d35L2229).